dspace-api/src/main/java/org/dspace/disseminate/DefaultCoverPageContributor.java

```diff
@@ -0,0 +1,62 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+
+package org.dspace.disseminate;
+
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import org.apache.tika.utils.StringUtils;
+import org.dspace.content.Item;
+import org.dspace.content.MetadataValue;
+
+public class DefaultCoverPageContributor implements CoverPageContributor {
+    @Override
+    public Map<String, String> processCoverPageParams(Item item, Map<String, String> parameters) {
+        // Title : Subtitle
+        var dcTitle = parameters.get("dc_title");
+        var dcTitleSubtitle = parameters.get("dc_title_subtitle");
+
+        if (StringUtils.isBlank(dcTitleSubtitle)) {
+            parameters.put("metadata_title", dcTitle);
+        } else {
+            parameters.put("metadata_title", "%s: %s".formatted(dcTitle, dcTitleSubtitle));
```

Comment:
better replace `formatted` method (added in Java 15) by `String.format("%s: %s", dcTitle, dcTitleSubtitle)` - this will allow to back-port this PR to DSpace v7.x (which is tested against Java 11 and does not require Java 15 as minimum requirement)